.drone.yml

```diff
@@ -14,6 +14,16 @@ trigger:
       - package.json
 
 steps:
+  - name: install_dependencies
+    image: node:18
+    commands:
+      - apt-get update
+      - apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgbm1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libnss3 lsb-release xdg-utils wget
+      - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+      - sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
+      - apt-get update
+      - apt-get install -y google-chrome-stable
```

Comment:
woah, i wonder if there's a clear way of doing this. i would imagine that there's a more straightforward way to do puppeteer in a linux box, but certainly could be wrong